doc/specs/vulkan/chapters/VK_KHR_swapchain/wsi.txt

```diff
@@ -147,23 +148,23 @@ minimized state before a sname:VkSwapchainKHR can be created, because:
   * pname:imageArrayLayers is the number of views in a multiview/stereo
     surface.
     For non-stereoscopic-3D applications, this value is 1.
-  * pname:imageUsage is a bitmask of elink:VkImageUsageFlagBits, indicating
-    how the application will use the swapchain's presentable images.
-  * pname:imageSharingMode is the sharing mode used for the images of the
+  * pname:imageUsage is a bitmask of elink:VkImageUsageFlagBits describing
+    the intended usage of the (acquired) swapchain images.
+  * pname:imageSharingMode is the sharing mode used for the image(s) of the
     swapchain.
   * pname:queueFamilyIndexCount is the number of queue families having
-    access to the images of the swapchain in case pname:imageSharingMode is
-    ename:VK_SHARING_MODE_CONCURRENT.
+    access to the image(s) of the swapchain in case pname:imageSharingMode
```

Comment:
nit: existing but probably should be simply "when" instead of "in case". Just weird phrasing...